tests/test_ipython_magics.py

```diff
@@ -123,69 +124,116 @@ def test_ignore_cells(
     assert validate(out, test_nb_path)
 
 
-def _indented_magics_toml_input() -> List[Tuple[str, Callable[..., bool], str]]:
-    """Input for handling indented magics test case using toml configuration."""
-    toml_config_file = "pyproject.toml"
+def _validate_magics_with_black(before: List[str], after: List[str]) -> bool:
+    """
+    Validate the state of the notebook before and after running nbqa with black.
+
+    Parameters
+    ----------
+    before
+        Notebook contents before running nbqa with autoflake
+    after
+        Notebook contents after running nbqa with autoflake
```

Comment:
I'm a bit confused here - the summary says "before and after running nbqa with black", but the param descriptions refer to autoflake

Reply:
Good catch. copy pasted. Forgot to change. I will change it to black.

Reply:
cool, feel free to merge after that

Reply:
Sure